<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TftpWidget</class>
 <widget class="QWidget" name="TftpWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="stateLab">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>311</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>请开始</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>311</width>
     <height>25</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">  text-align:right;</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>311</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openBtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateBtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>升级</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>进度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>状态</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportBtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
